packages/cozy-clisk/src/launcher/addData.js

```diff
@@ -12,7 +11,7 @@ const log = Minilog('addData')
  * @param {Array} entries : an array of objects corresponding to the data you want to save in the cozy
  * @param {string} doctype : the doctype where you want to save data (ex: 'io.cozy.bills')
  * @param {object} options : options object
- * @param {CozyClient} options.client : CozyClient instance
+ * @param {import('cozy-client/types/CozyClient').default} options.client CozyClient instance
```

Comment:
About this change in addData, and other files.
I have build and linked correctly to flagship app the lib.
I confirm it's working now with the konnector template. And i can delete addData, hydrateAndFilter, saveFiles, saveBills, saveIdentity